apps/picsa-tools/budget-tool/src/app/components/budget-summary/budget-summary.component.scss

```diff
@@ -0,0 +1,29 @@
+.summary-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin-bottom: 1.2rem;
+
+  h2 {
+    margin: 0 0 0.5rem 0;
+    padding: 0;
+    font-size: 1rem;
+    color: #000;
```

Comment:
nit(non-blocking)
You can probably just use the inherited color and remove this css. 
Also if you want to use a h2 element you should leave the font size for consistency across the app. If the font size is too big consider a h3 or h4 element instead, or just using a div with font-weight bold